bin/development

```diff
@@ -1,6 +1,7 @@
 #!/usr/bin/env ruby
 
-require File.join(File.dirname(__FILE__), '..', 'lib', 'parity')
+$LOAD_PATH.unshift File.expand_path(File.join("..", "..", "lib"), __FILE__)
+require "parity"
```

Comment:
It used to be procedure to put parity in your Gemfile, so while it's no longer recommended, I worry that there may be lots of apps in the wild with optimistic Gemfile declarations that use it.